### ISSUE TITLE
git prune-branches: does not pull updates for the main branch

### ISSUE THREAD

**Issue by kevgo** [Originate/git-town#324]
#### steps to reproduce
* I have a feature branch
* somebody else merges my feature branch (i.e. the main branch has remote updates that merge feature branches)
* I run `git prune-branches`

#### expected result
* it prunes my feature branch

#### actual result
* it does nothing

#### reason
* `git prune-branches` doesn't pull updates for the main branch, and therefore doesn't see that the feature branch is stale now

**Comment by kevgo** [comment]
This isn't possible with the current architecture, since it requires to perform a pull before we can create the step list. Given that this is an extreme edge case that can easily be circumvented by a `git sync`, I'm closing this issue as `wontfix`.
